pubspec.yaml

```diff
@@ -5,24 +5,25 @@ homepage: https://github.com/Workiva/over_react/
 authors:
   - Workiva UI Platform Team <uip@workiva.com>
 environment:
-  sdk: ">=1.21.1"
+  sdk: ">=1.23.0"
 dependencies:
-  analyzer: ">=0.26.1+3 <0.31.0"
+  analyzer: ">=0.30.0 <0.31.0"
   barback: "^0.15.0"
   js: "^0.6.0"
   meta: "^1.0.4"
   path: "^1.4.1"
   react: "^3.1.0"
   source_span: "^1.2.0"
   transformer_utils: "^0.1.1"
-  w_flux: "^2.5.0"
+  w_flux: "^2.7.1"
   platform_detect: "^1.1.1"
   quiver: ">=0.21.4 <0.25.0"
 dev_dependencies:
   browser_detect: "^1.0.4"
   matcher: ">=0.11.0 <0.13.0"
   coverage: "^0.7.2"
-  dart_dev: "^1.0.5"
+  dart_dev: "^1.7.6"
+  dart_style: ">=1.0.6 <2.0.0"
```

Comment:
mmm... must've added it when i was trying to get past the version solver issues w/ analyzer - which ended up being the `w_flux` package... will remove.